module/VuFind/tests/integration-tests/src/VuFindTest/Mink/ListViewsTest.php

```diff
@@ -94,6 +94,9 @@ protected function gotoRecord()
      *
      * @retryCallback tearDownAfterClass
      *
+     * @skip_html_validation true
+     * @todo                 Enable HTML validation when the issues are fixed in the upstream code
```

Comment:
Do you plan to open a JIRA ticket to cover this TODO?

Reply:
Yes, I opened https://openlibraryfoundation.atlassian.net/browse/VUFIND-1606 before posting the PR. :)

Reply:
No, sorry, that was for the other, and for this I failed to click the Create button. Now done as https://openlibraryfoundation.atlassian.net/browse/VUFIND-1607.